torch/_torch_docs.py

```diff
@@ -8615,7 +8539,7 @@ def merge_dicts(*dicts):
 Example::
 
     >>> a = torch.zeros(8, 8)
-    >>> b = torch.ones(8)
```

Comment:
Previously this would fail with the following error.
```
Failed example:
    a.slice_scatter(b, start=6)
Exception raised:
    Traceback (most recent call last):
      File "/home/oscar/miniconda3/envs/pytorch-deps/lib/python3.9/doctest.py", line 1336, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest torch.slice_scatter[3]>", line 1, in <module>
        a.slice_scatter(b, start=6)
    RuntimeError: expected src to have a size equal to the slice of self. src size = [8], slice size = [2, 8]
```
I don't know whether this is intended behaviour or not. Is `b` meant to broadcast into `a[6:, :]`?